apps/minaintyg/src/pages/Certificates/components/CertificateListFilter/CertificateStatusFilter.tsx

```diff
@@ -0,0 +1,23 @@
+import { IDSSelect } from '@frontend/ids-react-ts'
+import { useId } from 'react'
+import { CertificateStatus } from '../../../../schema/certificateList.schema'
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks'
+import { update } from '../../../../store/slice/certificateFilter.slice'
+
+export function CertificateStatusFilter({ options }: { options: CertificateStatus[] }) {
+  const id = useId()
+  const { statuses } = useAppSelector((state) => state.certificateFilter)
+  const dispatch = useAppDispatch()
+
+  return (
+    <IDSSelect>
```

Comment:
Can the select filters be made into a common component? There are a lot of duplicated code looking at the components and the tests

Reply:
I moved the general `Select` component from rehabstod into `@frontend/components` together with its Tooltip dependency, this way both rehabstod and minaintyg can share that component and removes a lot of the duplication. Each filter is similar but updates different parts of the store, has different labels etc. so they all need to be tested individually